code/game/objects/items/cardboard_cutouts.dm

```diff
@@ -0,0 +1,184 @@
+	visible_message("<span class='danger'>[src] has been hit by [P]!</span>")
+	playsound(src, 'sound/weapons/slice.ogg', 50, 1)
+	if(prob(P.damage))
+		push_over()
+
+/obj/item/cardboard_cutout/proc/change_appearance(obj/item/toy/crayon/crayon, mob/living/user)
+	if(!crayon || !user)
+		return
+	if(pushed_over)
+		to_chat(user, "<span class='warning'>Right [src] first!</span>")
+		return
+	var/new_appearance = input(user, "Choose a new appearance for [src].", "26th Century Deception") as null|anything in possible_appearances
+	if(!new_appearance || !crayon)
+		return
+	if(!do_after(user, 10, FALSE, src, TRUE))
+		return
+	user.visible_message("<span class='notice'>[user] gives [src] a new look.</span>", "<span class='notice'>Voila! You give [src] a new look.</span>")
+	alpha = 255
+	icon = initial(icon)
+	if(!deceptive)
+		color = "#FFD7A7"
+	switch(new_appearance)
+		if("Assistant")
+			name = "[pick(first_names_male)] [pick(last_names)]"
+			desc = "A cardboard cutout of an assistant."
+			icon_state = "cutout_greytide"
+		if("Clown")
+			name = pick(clown_names)
+			desc = "A cardboard cutout of a clown. You get the feeling that it should be in a corner."
+			icon_state = "cutout_clown"
+		if("Mime")
+			name = pick(mime_names)
+			desc = "...(A cardboard cutout of a mime.)"
+			icon_state = "cutout_mime"
+		if("Traitor")
+			name = "[pick("Unknown", "Captain")]"
+			desc = "A cardboard cutout of a traitor."
+			icon_state = "cutout_traitor"
+		if("Nuke Op")
+			name = "[pick("Unknown", "COMMS", "Telecomms", "AI", "stealthy op", "STEALTH", "sneakybeaky", "MEDIC", "Medic")]"
+			desc = "A cardboard cutout of a nuclear operative."
+			icon_state = "cutout_fluke"
+		if("Cultist")
+			name = "Unknown"
+			desc = "A cardboard cutout of a cultist."
+			icon_state = "cutout_cultist"
+		if("Clockwork Cultist")
+			name = "[random_name(pick(MALE,FEMALE))]"
+			desc = "A cardboard cutout of a servant of Ratvar."
+			icon_state = "cutout_servant"
+		if("Revolutionary")
+			name = "Unknown"
+			desc = "A cardboard cutout of a revolutionary."
+			icon_state = "cutout_viva"
+		if("Wizard")
+			name = "[pick(wizard_first)], [pick(wizard_second)]"
+			desc = "A cardboard cutout of a wizard."
+			icon_state = "cutout_wizard"
+		if("Shadowling")
+			name = "Unknown"
+			desc = "A cardboard cutout of a shadowling."
+			icon_state = "cutout_shadowling"
+		if("Xenomorph")
+			name = "alien hunter ([rand(1, 999)])"
+			desc = "A cardboard cutout of a xenomorph."
+			icon_state = "cutout_fukken_xeno"
+			if(prob(25))
+				alpha = 75 //Spooky sneaking!
+		if("Swarmer")
+			name = "Swarmer ([rand(1, 999)])"
+			desc = "A cardboard cutout of a swarmer."
+			icon_state = "cutout_swarmer"
+		if("Ash Walker")
+			name = random_name(pick(MALE,FEMALE),"Unathi")
+			desc = "A cardboard cutout of an ash walker."
+			icon_state = "cutout_free_antag"
+		if("Commando")
+			name = pick(commando_names)
+			desc = "A cardboard cutout of a strange commando."
```

Comment:
since death squads Don't Exist :tm: - probably shouldn't change it to your suggestion, @Fox-McCloud ...